client/payments/payment-recommendations.tsx

```diff
@@ -221,9 +215,7 @@ const PaymentRecommendations: React.FC = () => {
 					/>
 				</div>
 			</CardHeader>
-			<CardBody>
-				<List items={ pluginsList } />
-			</CardBody>
+			<List items={ pluginsList } />
 			<CardFooter>
```

Comment:
One last thing, the new padding seems to override the footer as well, it defaults to `size-large`.
I think we can remove the CardHeader size, and change the Card size to `medium` instead of `large`.

After doing that you can also remove this line: https://github.com/woocommerce/woocommerce-admin/blob/fa9ed31702e4f104844d8d40f368f84a79a196b9/client/payments/payment-recommendations.scss#L42